src/pages/Home/HomePage.jsx

```diff
@@ -0,0 +1,23 @@
+import { Footer } from "../../Components/Footer/Footer";
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { BrandNameSection } from "./components/BrandCategory";
+import { CategorySection } from "./components/CategorySection";
+import { HeroSection } from "./components/HeroSection";
+import { InterestCategorySection } from "./components/InterestCategory";
+import { NewCategorySection } from "./components/NewCategorySection";
+
```

Comment:
Suggestion: You can import all the files in one single file

Reply:
I will do that

Reply:
done